test/evp_extra_test.c

```diff
@@ -863,30 +863,72 @@ static EVP_PKEY *load_example_hmac_key(void)
     return pkey;
 }
 
+static int test_EVP_set_config_properties(void)
+{
+    char *fetched_properties = NULL;
+    const char test_propq[] = "test.fizzbuzz=buzzfizz";
+    int res = 0;
+
+    fetched_properties = EVP_get1_default_properties(OSSL_LIB_CTX_get0_global_default());
+    if (!TEST_ptr(fetched_properties)
```

Comment:
You don't need this check.  `TEST_str_eq()` does a NULL check internally.
It doesn't hurt, so not going to mandate this change.

Reply:
I consider NULL as an error - the correct value should be empty string so the check still makes sense

Reply:
That wasn't my point.  The `TEST_str_eq()` function does the NULL check for you.  This makes the `TEST_ptr()` check irrelevant.  A failure will get a decent diagnostic regardless.

I'd prefer to leave all of the checking to the `TEST_str_eq()` function but, as noted, I've approved it without.  Thus you doing nothing here is fine.

Reply:
Thanks for clarification!